package.json

```diff
@@ -159,14 +159,17 @@
     "tsconfig-paths-webpack-plugin": "^4.0.1",
     "typescript": "^5.0.4",
     "url-loader": "^4.1.1",
-    "vscode-extension-tester": "5.6.0",
+    "vscode-extension-tester": "^5.7.1",
```

Comment:
Upgraded this dependency as it has [bigger timeouts](https://github.com/redhat-developer/vscode-extension-tester/commit/9d472b09844c6000e45d064a4b38835842d5aebe#diff-f6d013860802040afba4b358dffd5b65468f48aa00377f49d00636bac2da904bR79)

[Changelog](https://github.com/redhat-developer/vscode-extension-tester/compare/v5.6.0...v5.7.1)